src/vs/workbench/contrib/welcome/page/browser/welcomePage.css

```diff
@@ -38,6 +38,11 @@
 	overflow: hidden;
 }
 
+/* {{SQL CARBON EDIT}} Overwrite ADS caption display to be initial instead of block */
+.monaco-workbench .part.editor > .content .welcomePage .row .splash .showOnStartup .caption {
```

Comment:
why is this necessary?

Reply:
Without this, the checkbox and the "Show welcome page on startup" string are on different lines. Was just trying to scope this down. If you have any other suggestions, more than happy to  entertain them.

Reply:
I would be interested in why thats the case for ours but not theirs.

Reply:
In sql/base/browser/ui/table/media/table.css we say that every label has display:block, so that's why there's a difference between ADS and VSCode

Reply:
We should update that css to be more scoped to table then

Reply:
For sure. But changing that could have unintended consequences, especially this late in the release.

Reply:
The problem is if this goes into master, the likely hood that we will do the proper fix later is pretty low (based on the past). Maybe a better fix would be scope table in this PR and if you are worried about that affecting release, do a release branch only fix that removes that and adds this?

Reply:
We can always take a different commit in release/1.6 then in master if we want to make different fixes in each branch.

Reply:
I've fixed this. Looked at tables and compared screenshots for results; couldn't find a difference. Removed the edit in welcomePage.css.